index.js

```diff
@@ -160,6 +160,12 @@ Hbs.prototype.createRenderer = function() {
     var tplPath = hbs.getTemplatePath(tpl),
       template, rawTemplate, layoutTemplate;
 
+    if(tplPath === undefined) {
+      this.status = 500;
```

Comment:
@shellscape thank you for starting a discussion about this. I looked into this a bit more, and I agree with your suggestion. #42 is the tracking issue to utilize the standard koa error handling mechanisms.